### PR TITLE
Add missing operations (and translate them). Modify some data types.

### PR DESCRIPTION
**Pull request by gustavoaca1997**

### CODE REVIEW

**Comment by gustavoaca1997** [comment]
What is in this PR?
- Translate names to English
- Add missing operations like `IfElse`. I think `NewLabel` is not necessary. I can remove it if you all want to. We have to decide what to do with `Cast`.
- Add `Label Int` as a kind of `Operand`.
- `Constant b` is now `Constant (String, b)`, where the string corresponds to the token itself; `b` is the type of the constant (`int`, `float`, `char`, etc.).
- Add some implementations for `show` for each instruction. There are some missing implementations of course. A `<?>` is gonna be shown if the operation has no implementation for `show`.
- I considered `rValue2` from `if`, `ifElse` and `goto` as the label to jump to. Do you all like that decission? It is good to choose the same field for the label because it will make easier to code.

**Review by german1608 — APPROVED** [review]

**Review by ShonTitor — APPROVED** [review]

File: TACType.hs [hunk 1]
```diff
@@ -3,31 +3,108 @@ module TACType where
...
+    -- | Greater than or equal
+    Gte        |
+    -- | Less than
+    Lt           |
+    -- | Less than or equal
+    Lte        |
+    -- | Equal
+    Eq           |
+    -- | Not equal
+    Neq         |
```
Comment (ShonTitor) [inline TACType.hs:79]: Creo que todos estos deberían ser instrucciones de salto

**Review by Naty-S — APPROVED** [review]
Considerar cambiar los comparadores a instrucciones de salto

**Comment by Naty-S** [comment]
Considerar cambiar los comparadores a instrucciones de salto (hablarlo)

**Pull request merged**